### PR TITLE
fix(docs-ui): limit list maxLevel to 3 when in table & set line-through style on checklist

### PR DESCRIPTION
**Pull request by weird94**
close https://github.com/dream-num/univer-pro/issues/2073

## Pull Request Checklist

- [ ] Related tickets or issues have been linked in the PR description (or missing issue).
- [ ] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [ ] Unit tests have been added for the changes (if applicable).
- [ ] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

File: packages/docs/src/basics/paragraph.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ICustomTable, IParagraph } from '@univerjs/core';
+
+export function isParagraphInTable(paragraph: IParagraph, tables: ICustomTable[]) {
```
Comment (wzhudev) [inline packages/docs/src/basics/paragraph.ts:19]:
```suggestion
export function hasParagraphInTable(paragraph: IParagraph, tables: ICustomTable[]) {
```

Comment (wzhudev) [reply packages/docs/src/basics/paragraph.ts:19]: `isXXX` is usually considered as a property  or an variable instead of a function.

**Review by wzhudev — CHANGES_REQUESTED** [review]
It is recommended to use a correct syntax.

**Comment by oumomomo** [comment]
使用多级列表后需要删除两次才可以成功删除单元格内的换行符
操作步骤：
1、新增表格，在单元格内输入字符（需要在单元格内留出一个字符的空间）
image
2、使用多级列表直到单元格内会展示换行符
3、删除换行符
image


实际表现：需要删除两次才可以删除换行符

**Comment by oumomomo** [comment]
在单元格内不停增加多级列表会失焦，失焦后选中内容删除表格白屏

1、在单元格输入内容使用多级列表直到失焦
2、选中的单元格内容删除（第一次无反应可以多按两次删除键）
[2024_8_17 11_08_47 video.webm](https://github.com/user-attachments/assets/2ea598b2-bef8-4344-aa14-816c45c1866e)
image

**Comment by oumomomo** [comment]
在单元格内使用第三个有序列表时会失焦

操作步骤：
1、插入表格并在单元格输入内容
2、使用有序列表内的第三个
3、选中内容不停增加子列表
image
[2024_8_17 11_34_23 video.webm](https://github.com/user-attachments/assets/fa69e922-0059-469b-8652-7c15dc777856)

**Review by wzhudev — APPROVED** [review]
LGTM

**Pull request merged**